### ISSUE TITLE
Odd verbage hard to parse 

### ISSUE THREAD

**Issue by nadalin** [w3c-ccg/did-wg-charter#60]
"DIDs are URLs that relate a DID subject to means for trustable interactions with that subject"

I'm not sure about this sentence as it would be hard to parse for non-native English speakers

**Comment by TallTed** [comment]
Do you have any suggestions for rewording?

**Comment by nadalin** [comment]
DIDs are URLs that relate a DID subject for trustable interactions with that subject

**Comment by TallTed** [comment]
> DIDs are URLs that relate a DID subject for trustable interactions with that subject

This does not seem to resolve your stated concern.  I am a native English speaker, and I cannot parse this, which appears to be not so much a rewording of the original as a removal of two words ("to means") which are key to the original sentence's meaning.  I cannot imagine that a non-native English speaker would be able to parse it any better.  Any other suggestions?

**Comment by nadalin** [comment]
I don't understand the "to means" please explain how that applies and then I may be able to suggest alternative text

**Comment by msporny** [comment]
@nadalin, what about this as a replacement?

"DIDs are URLs that relate a DID subject to one or more mechanisms that may be used for trustable interactions with that subject."

**Comment by talltree** [comment]
I'm fine with that.
